### PR TITLE
zhCN/zhTW update.

### PR DESCRIPTION
**Pull request by adavak**

### CODE REVIEW

File: BlackwingLair/Locales/zhCN.lua [hunk 1]
```diff
@@ -18,9 +18,8 @@ if L then
 	L.breath = "吐息警报"
 	L.breath_desc = "吐息警报"
 
-	L.debuffs = "{23174} （{605}）" -- 多彩变形 （精神控制）
```
Comment (funkydude) [inline BlackwingLair/Locales/zhCN.lua]: The reason I added this was so that you could have the correct brackets `（）`

You don't want that?

Comment (adavak) [inline BlackwingLair/Locales/zhCN.lua]: that's Chinese () not need to do. very small. nobody focus it.

Comment (funkydude) [inline BlackwingLair/Locales/zhCN.lua]: My point is that there is no harm in having this code. It needs to be in the locale anyway, so why not make it proper Chinese? :)

Comment (adavak) [inline BlackwingLair/Locales/zhCN.lua]: attention to detail is good.
i make it again.

**Pull request merged**